specification/resources/resource-manager/Microsoft.Solutions/stable/2019-03-01/managedapplications.json

```diff
@@ -0,0 +1,1885 @@
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationDefinition"
+          },
+          "description": "The array of managed application definitions."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "description": "List of managed application definitions."
+    },
+    "ProvisioningState": {
+      "description": "Provisioning status of the managed application.",
+      "readOnly": true,
+      "enum": [
+        "Accepted",
+        "Running",
+        "Ready",
+        "Creating",
+        "Created",
+        "Deleting",
+        "Deleted",
+        "Canceled",
+        "Failed",
+        "Succeeded",
+        "Updating"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "ApplicationLockLevel": {
+      "description": "The managed application lock level.",
+      "enum": [
+        "CanNotDelete",
+        "ReadOnly",
+        "None"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ApplicationLockLevel",
+        "modelAsString": false
+      }
+    },
+    "ApplicationArtifactType": {
+      "description": "The managed application artifact type.",
+      "enum": [
+        "Template",
+        "Custom"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ApplicationArtifactType",
+        "modelAsString": false
+      }
+    },
+    "ApplicationArtifact": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The managed application artifact name."
+        },
+        "uri": {
+          "type": "string",
+          "description": "The managed application artifact blob uri."
+        },
+        "type": {
+          "$ref": "#/definitions/ApplicationArtifactType",
+          "description": "The managed application artifact type."
+        }
+      },
+      "description": "Managed application artifact."
+    },
+    "ApplicationProviderAuthorization": {
+      "properties": {
+        "principalId": {
+          "type": "string",
+          "description": "The provider's principal identifier. This is the identity that the provider will use to call ARM to manage the managed application resources."
+        },
+        "roleDefinitionId": {
+          "type": "string",
+          "description": "The provider's role definition identifier. This role will define all the permissions that the provider must have on the managed application's container resource group. This role definition cannot have permission to delete the resource group."
+        }
+      },
+      "required": [
+        "principalId",
+        "roleDefinitionId"
+      ],
+      "description": "The managed application provider authorization."
+    },
+    "ErrorResponse": {
```

Comment:
This does not appear to align to RPC error contract, see: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content

Reply:
This actually comes with original schema, not changed by this PR.